### PR TITLE
Add discourse integration

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

#### Additional deploy notes

This devops pr needs to be merged: https://github.com/santiment/devops/pull/273

[//]: # (#### Screenshots)

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -8,6 +8,8 @@ use Mix.Config
 # General application configuration
 config :sanbase, ecto_repos: [Sanbase.Repo]
 
+config :sanbase, Sanbase, url: {:system, "SANBASE_URL", "https://sanbase-low-stage.santiment.net"}
```
Comment (tspenov) [inline config/config.exs]: can't we use `website_url` which is further down in config

File: lib/sanbase/discourse/api.ex [hunk 2]
```diff
@@ -0,0 +1,62 @@
+defmodule Sanbase.Discourse.Api do
+  require Logger
+  require Mockery.Macro
+
+  alias Sanbase.Discourse.Config
+
+  def publish(title, text, url \\ Config.discourse_url()) do
+    handle_request(
```
Comment (tspenov) [inline lib/sanbase/discourse/api.ex]: Do you think it is more readable like this:
```elixir
url
|> publish_topic(title, text)
|> handle_request()
```

File: lib/sanbase/voting/post.ex [hunk 3]
```diff
@@ -52,7 +53,7 @@ defmodule Sanbase.Voting.Post do
 
   def update_changeset(%Post{} = post, attrs) do
     post
-    |> cast(attrs, [:title, :short_desc, :link, :text])
+    |> cast(attrs, [:title, :short_desc, :link, :text, :discourse_topic_url])
```
Comment (tspenov) [inline lib/sanbase/voting/post.ex]: Do we need to cast `:discourse_topic_url` in create and update changesets ?

Comment (IvanIvanoff) [reply lib/sanbase/voting/post.ex]: Not really.

**Pull request merged**